### DISCUSSION TITLE
How to launch weasis from my WEB

### DISCUSSION THREAD

**Discussion by AdolfoBanchio** [nroduit/Weasis#379] [Q&A]
I have a web where I have medical studies form patients and I have them saved in my own server.  I want to have an option to launch weasis directly from the web with the specified study archive of the patient. 

I tried using this https://nroduit.github.io/en/getting-started/weasis-protocol/index.html#how-to-build-an-uri

but it does not work.

in the files of my webpage i have a /estudios directory where al the files of the studies are. Is there any way of launchig weasis and open a file from my server?

I do not have a PACS system or similar

**Comment by nroduit** [comment]
A URL cannot be a directory. It is a resource that corresponds to a file.

See https://github.com/nroduit/Weasis/issues/353#issuecomment-1327912289
